storybook/src/components/Button/Button.docs.mdx

```diff
@@ -16,28 +16,45 @@ import README from "../../../../packages/css/src/components/button/README.md?raw
 
 ### Primary
 
-The most important call-to-action.
-One primary Button may be used per screen.
+A primary button draws attention to the most important call to action.
+Only one primary Button should be used per screen or, in applications, large section.
```

Comment:
Application have screens as well, no? Why the exception for large sections here?

Reply:
As an example, each panel or drawer of an application could have a primary button. I assume the guideline to have max 1 per page is thought up from the perspective of a web page.

But it might be too early to mention this in docs.

Reply:
```suggestion
Only one primary Button should be used per screen.
```